mermithid/processors/misc/ComplexLineShape.py

```diff
@@ -0,0 +1,598 @@
+'''
+Fits data to complex lineshape model.
+Author: E. Machado, Y.-H. Sun, E. Novitski
+Date: 4/8/20
+'''
+
+from __future__ import absolute_import
+
+import numpy as np
+import scipy as sp
+from scipy.optimize import curve_fit
+from scipy import integrate , signal, interpolate
```

Comment:
you import scipy and all the functions from scipy that you use separately. then you use sometimes the function and sometimes sp.function. please remove either line 10 or line 12 and change the function calls accordingly.